yellowbrick/features/rankd.py

```diff
@@ -271,20 +452,8 @@ def draw(self, X, **kwargs):
         cb = self.ax.figure.colorbar(mesh, None, self.ax)
         cb.outline.set_linewidth(0)
 
-    def finalize(self, **kwargs):
-        """
-        Finalize executes any subclass-specific axes finalization steps.
-        The user calls poof and poof calls finalize.
-
-        Parameters
-        ----------
-        kwargs: dict
-            generic keyword arguments
+        # Reverse the rows to get the lower left triangle
+        self.ax.invert_yaxis()
 
-        """
-        # Set the title
-        self.set_title(
-            "{} Ranking of {} Features".format(
-                self.ranking_.title(), len(self.features_)
-            )
-        )
+        self.ax.set_xticklabels(self.features_, rotation=90, ha='left')
```

Comment:
As in the other case the feature names default to integers... I think that was a feature that you built!  :)